packages/office-ui-fabric-react/src/components/Button/CommandBarButton/CommandBarButton.styles.ts

```diff
@@ -9,7 +9,9 @@ export const getStyles = memoizeFunction(
     const baseButtonStyles: IButtonStyles = getBaseButtonStyles(theme);
     const baseSplitButtonStyles: IButtonStyles = getSplitButtonStyles(theme);
 
-    const { palette: p } = theme;
+    const { palette: p, semanticColors } = theme;
+    const BUTTON_ICON_CLASSNAME = '.ms-Button-icon';
+    const MENU_ICON_CLASSNAME = 'ms-Button-menuIcon';
```

Comment:
suggestion: you might wanna import the already exported const `ButtonGlobalClassNames` from `BaseButton.classNames.ts and assign the values to your local consts. This way you keep the source in one place.